meta-balena-common/conf/distro/include/balena-os.inc

```diff
@@ -67,8 +67,8 @@ INITRAMFS_TASK = ""
 # resinOS defaults to ipk packages
 PACKAGE_CLASSES ?= "package_ipk"
 
-# Default the docker storage driver to aufs
-BALENA_STORAGE ?= "aufs"
+# Default the docker storage driver to overlay2
+BALENA_STORAGE ?= "overlay2"
```

Comment:
need to check this PR works with devices that specify aufs (due to old kernels etc)

Reply:
What devices would that be? Intel Quark, ...? I'm not sure I have one test that myself here

Reply:
Rpi will have this support, won't it?
I'm looking forward to this change so I can build balenaOS on my Mac for my RaspberryPi device :)

Reply:
@robertgzr one of the odroids I think.

Reply:
We still need to verify this I think... does anybody from the devices team have the devices relevant to this?

cc @floion @horia-delicoti